### ISSUE TITLE
Very peculiar changeling bug where you're set into godmode and can never be healed 

### ISSUE THREAD

**Issue by Bawhoppen** [tgstation/-tg-station#11805]
I've had 2 instances of this bug happening in the past week, both under very different circumstances. 
The bug is essentially that I was in like the knocked out sleep kind of thing (not crit), where you're supposed to wake up after a few seconds or whatever but then I was never able to wake up.

The first time I was killed at roundstart by a ling, and then debrained and waited in robotics for awhile. They caught the ling dragged him while he knocked out by an n2o tank (like the surgery anestheia kind) and they debrained him, chucked my brain in his body then cloned me. I was then put into cryo and after I was fully healed they tried to shake me to wake me up but it was impossible and the admins online at the time didn't know what was up. After like 15 minutes they just deleted me and respawned me to fix it.

The second time I was the one who spawned as a ling, killed some people and absorbed some people for objectives, and I was running with the strained muscles but I had it on too long so I collapsed of exhaustion and never woke up. MSO was on at the time and he tried to figure it out, and figured out I was somehow stuck in godmode so thats why he couldn't heal me. He eventually just respawned me aswell as the round was going to end.
These are the abilties I chose: 
Aug eyesight, Fleshmend, Mimic voice, Armblade, Anatomic Panacea, Strained muscles, Engorged chem glands.

tl;dr Its a weird bug where you get stuck in godmode (sleep) that is likely related to changelings.

**Comment by Dorsisdwarf** [comment]
@Bawhoppen 

Please try and reproduce this, this sounds like something's going badly wrong somewhere

**Comment by Dorsisdwarf** [comment]
Could not reproduce 25/09/2015

**Comment by Bawhoppen** [comment]
Yea, its weird on when it happened. I'll try and do some things I did to see if I can reproduce it by luck.
